draft-ietf-taps-arch.md

```diff
@@ -487,9 +487,9 @@ There are several reasons, however, that an application might want to explicitly
 - Privacy concerns about allowing Connections to multiplex together, which can tell a Remote Endpoint that all of the Connections are coming from the same application (for example, when Connections are multiplexed HTTP/2 or QUIC streams).
 - Performance concerns about Connections introducing head-of-line blocking due to multiplexing or needing to share state on a single thread.
 
-The Transport Services API SHOULD allow applications to explicitly define Connection Groups that force separation of Cached State and Protocol Stacks. For example, a web browser application might use Connection Groups with separate caches for different tabs in the browser to decrease linkability.
```

Comment:
I actually prefer the change proposed to use /can/ .... although others may protest? -  I think that whether a TAPS system /implements/ this is not fundamental to TAPS.

Reply:
let's pick this up at/after 107.